code/modules/mob/abstract/new_player/character_traits.dm

```diff
@@ -79,3 +79,87 @@
 	name = "Major Hemophilia"
 	desc = "Your blood lacks ALL clotting factors, causing wounds to never stop bleeding."
 	trait_type = TRAIT_DISABILITY_HEMOPHILIA_MAJOR
+
+
+ABSTRACT_TYPE(/datum/character_disabilities/bruising)
+	name = "Bruised Limb"
+	var/affected_limb
```

Comment:
I would have wrote DMDoc but honestly it takes 2 neurons to understand what this does so